### ISSUE TITLE
Feature request: add link to source code on your site

### ISSUE THREAD

**Issue by raybellwaves** [matt-kruse/MyOpenLeaderboard.com#5]
There is a probably a few crossfit techies who would love to know how the site works. The link would direct them here and they may even contribute?

Could be next to the 'What's New?` link

**Comment by matt-kruse** [comment]
I like the idea of collaboration, but I don't necessarily want to draw attention here because I don't want multiple devs hitting my proxy service, which would overwhelm the crossfit servers and get me shut down. I've already had problems with that, so I had to introduce aggressive caching both in memory and in a database. So actually, most of the complex logic lies in AWS Lambda, not here.

If I could get a "test proxy" running that would shield my lambda function and the crossfit servers from any additional traffic, I wouldn't mind having other devs join in.

Also, my code for this is total garbage, thrown together and evolved as quickly as possible. I'm a bit embarassed by it, honestly. :) It's a hack on top of a hack! But that is actually something other devs could help with. You know, maybe it would be a good idea a month before the 2019 Open to get devs together on this project and see if we can release something even better next year.

**Comment by raybellwaves** [comment]
Fair enough. I understand. You could put out a call on r/crossfit one month before the next open for help. Although from my experience people say they will help but rarely do...

**Comment by matt-kruse** [comment]
Ha, that's been my experience as well. Or the people who DO want to help are not qualified to do so, and end up being more work to "maintain" than if they didn't "help" at all. :(
